src/public/media.ts

```diff
@@ -77,3 +92,268 @@ export function captureImage(callback: (error: SdkError, files: File[]) => void)
   const messageId = sendMessageRequestToParent('captureImage');
   GlobalVars.callbacks[messageId] = callback;
 }
+
+/**
+ * Media object returned by the select Media API
+ */
+export class Media extends File {
+  /**
+   * A preview of the file which is a lightweight representation.
+   * In case of images this will be a thumbnail/compressed image in base64 encoding.
+   */
+  public preview: string;
+
+  /**
+   * Gets the media in chunks irrespecitve of size, these chunks are assembled and sent back to the webapp as file/blob
+   * @param callback returns blob of media
+   */
+  public getMedia(callback: (error: SdkError, blob: Blob) => void): void {
```

Comment:
is this callback use only for failure case. We are thinking to support promise based sdk API in future. It will be better to update this api public getMedia(successCallback: (blob: Blob) => void, failureCallback: (error: SdkError) => void):

Reply:
After we introduced SdkError we had decided we will keep error as the 1st param of callback and remaining subsequently. When we move to promises we can do something like: Promise.resolve([error, data]). Right now in case of error we return error and in case of success we return blob.